src/pages/docs/en/other/faq.md

```diff
@@ -5,6 +5,26 @@ position: 0
 layout: "@docs"
 ---
 
+## Why is Unauthorized error thrown when I try to access my app?
+By default all apps are private. You can make the app or parts of it public by editing the Spacefile and adding the `public_routes` key. Read more about it [here](/docs/en/basics/micros#public-routes).
+
+## How do Micros work?
+Deta space is a serverless platform therefore it can't have long running servers for handling requests. Instead it creates a new process for each request, responds to it and then that process gets freed up. But the process has a timeout of 20 seconds. So if your request takes more than 20 seconds to complete, it will be terminated. 
+
+Lean more about Micros [here](/docs/en/basics/micros).
+
+## Do Micros support websockets?
+Because Micros are serverless compute instances with a 20 seconds maximum lifespan per request, they do not support websockets.
+
+## My Micro timed out, what do I do?
+Your request took longer than 20 seconds to finish if your Micro timed out. You can try to speed up your code by optimising it or by putting another solution in place that doesn't need a constant connection.
+## Support for Python 3.11?
+Current micros include Python runtime support up to 3.10 and are powered by AWS Lambda. Micros will therefore support it once AWS Lambda begins to support those runtime.
+
+## What are the limits of a Micro?
```

Comment:
```suggestion
## What are the limits of Micros?
```

Reply:
Was to signify that each "Micro" has those limits rather than all micros ( like 5 micros in your app )